recipes/llvm-core/all/conanfile.py

```diff
@@ -29,6 +29,8 @@
 
 required_conan_version = ">=1.62.0"
 
+LLVM_TARGETS = "AArch64;AMDGPU;ARM;AVR;BPF;Hexagon;Lanai;LoongArch;Mips;MSP430;NVPTX;PowerPC;RISCV;Sparc;SystemZ;VE;WebAssembly;X86;XCore"
```

Comment:
I recommend using a list for this variable instead of a special delimited string.

Reply:
Being a bit lazy here as just copied and pasted from the LLVM docs.